comments/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.conf.urls import url
+
+from comments.views import CommentViewSet
+
+comment_add = CommentViewSet.as_view(
+    {
+        "post": "create",
+    }
+)
+
+comments_list = CommentViewSet.as_view(
+    {
+        "get": "list",
+    }
+)
+
+comment_detail = CommentViewSet.as_view(
+    {
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
+)
+
+urlpatterns = [
+    url(r"^comments/add/$", comment_add, name="add-comment"),
```

Comment:
We don't need `add` postfix. Use `POST` to `r"^comments/$"`

Reply:
`r"^comments/$"` can be used to create a comment now